setup.cfg

```
@@ -29,14 +29,15 @@ classifiers =
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11 (Only for Django 4.1/4.2)
     Topic :: Software Development :: Libraries :: Python Modules
 
 [options]
 zip_safe = False
 include_package_data = True
 packages = find:
 install_requires =
-    django>=3.2
+    django>=4.2
```

3.2 is still supported... That's why we keep testing it.
```suggestion
    django>=3.2
```